docs/sources/tempo/shared/azure-metrics-generator.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
this whole file is incorrect and not needed

Reply:
it's for Helm. I'll clarify in the file.